RELEASE.md

```diff
@@ -162,8 +162,65 @@ Tagging a release candidate will trigger CI to publish to NPM, dockerhub, and Gi
   - `git tag -am "v1.1.1-rc.0" v1.1.1-rc.0`
   - `git push origin v1.1.1-rc.0`
 
-Continue following the "test release candidate" and "merge release candidate" sections. Testing window may be modified depending on the severity of the bug fixed.
+### 3. Test hotfix release candidate
 
+Once a hotfix release candidate is created, the Lodestar team may begin a modified hotfix testing period consisting of a quick sanity check or longer if required.
+
+If the hotfix does not address the purpose of the hotfix release, or there is another bug discovered during this modified hotfix testing period which significantly impacts performance, security, or stability, and it is determined that it is no longer prudent to promote the `rc.x` candidate to `stable`, then it will await an additional fix by the team. The fix will be committed to `unstable` first, then cherrypicked into the `rc/v1.1.1` hotfix branch. Then we publish and promote the new commit to `rc.x+1`. The modified hotfix testing period will reset.
+
+For example: After modified hotfix testing period, is the original bug resolved? Is performance equal to or better than latest stable?
+
+- **Yes**: Continue to the next release step
+- **No**: If it a small issue fixable quickly with another hot-fix?
+  - **Yes**: Merge fix(es) to `unstable`, push the fix(es) to `rc/v1.1.1` hotfix branch, go to step 2, incrementing the rc version
+  - **No**: Abort the release. Close the `chore: v1.1.v release` PR, delete the branch, and start the whole release process over.
+
+### 4. Merge hotfix release candidate
+
+- Ensure step 3 testing is successful and there is sufficient consensus to release `v1.1.1`.
+- Approving the `chore: v1.1.1 release` PR means a team member marks the release as safe, after personally reviewing and / or testing it.
+- Merge `chore: v1.1.1 release` PR to stable **with "merge commit"** strategy to preserve all history.
+- Merge stable `stable` into `unstable` **with merge commit** strategy. Due to branch protections in `unstable` must open a PR. If there are conflicts, those must be resolved manually. Gitflow may cause changes that conflict between stable and unstable, for example due to a hotfix that is backported. If that happens, disable branch protections in unstable, merge locally fixing conflicts, run lint + tests, push, and re-enable branch protections. See "Backporting merge conflicts from stable to unstable".
+
+#### Backporting merge conflicts from stable to unstable
```

Comment:
This section applies to both hotfix releases and normal releases. Consider moving to an appendix or duplicating for both lists.

Reply:
I considered this, but instead of jumping back and forth between different parts of the doc for a minor bump vs hotfix bump, it would be easier to lay it out the process sequentially, even if some of it is repetitive. This way if you're doing a hotfix bump, you can just stick to that section.